aio_azure_clients_toolbox/clients/service_bus.py

```diff
@@ -169,8 +176,16 @@ async def ready(self, conn: ServiceBusSender) -> bool:
             try:
                 await conn.schedule_messages(message, now)
                 return True
-            except (AttributeError, exceptions.AzureError):
-                logger.warning(f"ServiceBus readiness check #{3 - attempts} failed; trying again.")
+            except (ServiceBusAuthorizationError, ServiceBusAuthenticationError):
+                # We do not believe these will improve with repeated tries
+                logger.error(
```

Comment:
Would `log.exception` be more useful for users?

Reply:
hmm.  Maybe it would. That's a fair point. I can switch it. 

(side note: I _never_ remember that `log.exception` exists... 😬 )